api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -284,7 +284,9 @@ message CommonResponse {
 
   // Instructions on how to manipulate the headers. When responding to an
   // HttpBody request, header mutations will only take effect if
-  // the current processing mode for the body is BUFFERED.
+  // the current processing mode for the body is: 1) BUFFERED; 2) or STREAMED and
+  // the below filter configuration is enabled:
+  // [#comment:TODO :ref:`send_body_without_waiting_for_header_response <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExternalProcessor.send_body_without_waiting_for_header_response>`]
```

Comment:
```suggestion
  // the current processing mode for the body is BUFFERED.
  // [#comment:TODO(yanjunxiang-google) rephrase last sentence once send_body_without_waiting_for_header_response is not hidden:
  // the current processing mode for the body is: 1) BUFFERED; 2) or STREAMED and
  // the :ref:`send_body_without_waiting_for_header_response <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExternalProcessor.send_body_without_waiting_for_header_response>` is enabled.]
```

Otherwise the documentation seems broken (see https://storage.googleapis.com/envoy-pr/4f092d2/docs/api-v3/service/ext_proc/v3/external_processor.proto.html#service-ext-proc-v3-commonresponse)